Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,39 @@
+class Pgroonga < Formula
+  desc "PostgreSQL plugin to use Groonga as index"
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
+  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
+
+  depends_on "pkg-config" => :build
+  depends_on "groonga"
+  depends_on "postgresql"
+
+  def install
+    # To find HOMEBREW_PREFIX/bin/pg_config
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
```

Comment:
I still think it's a bad idea and should be avoided at most costs. If they need to share a common permanent prefix I'd almost rather dump `postgresql` related binaries and plugins in `HOMEBREW_PREFIX/var/postgresql_bin/*` and then hardlink or copy over them with the newer versions `postinstall` as necessary.

People would have to add that directory to their `$PATH` but I think it's less likely to randomly break than circumventing a fairly big part of superenv.